buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijVersionHelper.kt

```diff
@@ -54,4 +54,18 @@ object IntellijVersionHelper {
   private fun getFile(): File {
     return File("intellijVersions.properties")
   }
+
+  infix fun String.versionIsNewerThan(rhsBuildNumber: String): Boolean {
```

Comment:
Actually! to work around the problem with Strings sometimes keeping `222.1234.56`-style build numbers, and sometimes `2022.2`-style versions, I'd consider using _newtypes_, called _inline classes_ in Kotlin: https://kotlinlang.org/docs/inline-classes.html#calling-from-java-code

This way, an attempt to pass a build number where version is expected (or vice versa) will be detected at Kotlin compile time.

Can be done on a separate PR

Reply:
Also, comparision method can be called as `isNewerThan`, no need to specify `version`/`buildNumber` in method name, as it'll be obvious from operand types

Reply:
Wouldn't that be an overkill? We don't have that many uses for version/build numbers. The only place where there could be a problem right now is the [Release repo](https://www.jetbrains.com/intellij-repository/releases/) where both version numbers and build numbers can be found, right now we rely simply on the fact that version numbers are always higher(newer) than build numbers.

Reply:
Okay, leaving that up to you then